tests/test_assets.py

```diff
@@ -591,14 +591,14 @@ def test_consume_asset_as_identifier(self):
     def test_sid_assignment(self):
 
         # This metadata does not contain SIDs
-        metadata = {'PLAY': {'symbol': 'PLAY'},
-                    'MSFT': {'symbol': 'MSFT'}}
+        metadata = {'PLAY': {'symbol': 'PLAY', 'sid': 1},
```

Comment:
@jfkirk isn't the point of this test to exercise how we handle inputs that **don't** provide sid data?